cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -73,20 +73,20 @@ def __init__(self, scale, variant):
         references = {
             'maint': {
                 'small': {
-                    'dom:gpu': {'perf': (29.3, -0.05, None, 'ns/day')},
-                    'daint:gpu': {'perf': (30.3, -0.10, None, 'ns/day')}
+                    'dom:gpu': {'perf': (0.0, -0.05, None, 'ns/day')},
+                    'daint:gpu': {'perf': (0.0, -0.10, None, 'ns/day')}
                 },
                 'large': {
-                    'daint:gpu': {'perf': (42.0, -0.10, None, 'ns/day')}
+                    'daint:gpu': {'perf': (0.0, -0.10, None, 'ns/day')}
```

Comment:
I can understand the comment about Dom, but not for this one. Why `0.0` also here?